swift/ios-shoppe-demo/Views/CheckoutTableViewController.swift

```diff
@@ -46,14 +46,17 @@ class CheckoutTableViewController: UITableViewController {
             productView.modalPresentationStyle = .formSheet
             productView.checkoutTableViewController = self
 
-            // MARK: - FullStory Example CustomEvent
-            fsCreateEvent(event: .checkout, with: order.orderSummary())
+            // MARK: - FullStory Example Custom Event "Order Completed" for conversions: revenue attribution
+            fsCheckoutSuccessEvent(event: .checkout, with: order)
 
             present(productView, animated: true, completion: nil)
         }
         else {
+            let msg = "User tried to checkout without confirmation."
+            let err: CheckoutError = CheckoutError(errorCode: "AC10XY2", message: msg, order: order)
```

Comment:
CheckoutError type currently lives in fsManager, and for now it's very specifically for fs events. In the future we may want to pull it into it's own model?